internal/guest/storage/scsi/scsi.go

```diff
@@ -262,7 +328,7 @@ func UnplugDevice(ctx context.Context, controller, lun uint8) (err error) {
 		trace.Int64Attribute("controller", int64(controller)),
 		trace.Int64Attribute("lun", int64(lun)))
 
-	scsiID := fmt.Sprintf("0:0:%d:%d", controller, lun)
+	scsiID := fmt.Sprintf("%d:0:0:%d", controller, lun)
```

Comment:
should this be a `scsiIDFmt`?

Reply:
Since this is only used at this location it should be okay to keep it as a string here. If we start using this somewhere else then we can move it to a const.